core/backend/src/SchemaSync.ts

```diff
@@ -0,0 +1,48 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+
+/** @packageDocumentation
+ * @module SQLiteDb
+ */
+
+import { CloudSqlite } from "./CloudSqlite";
+import { VersionedSqliteDb } from "./SQLiteDb";
+
+/** @beta */
+export namespace SchemaSync {
```

Comment:
Consider what if any benefit you're getting from putting these two classes and one `const` inside a namespace.
e.g., you're not even getting the debatable benefit of shortening the type names (`SchemaSync.SchemaSyncDb`?)
Consider removing namespace and making `defaultDbName` a static field of `SchemaSyncDb`.

Reply:
see my recent changes